crates/router/src/routes/app.rs

```diff
@@ -271,20 +271,23 @@ pub struct Customers;
 #[cfg(any(feature = "olap", feature = "oltp"))]
 impl Customers {
     pub fn server(state: AppState) -> Scope {
+        let mut root_resource = web::resource("");
+
         let mut route = web::scope("/customers").app_data(web::Data::new(state));
 
         #[cfg(feature = "olap")]
         {
+            root_resource = root_resource.route(web::get().to(customers_list));
```

Comment:
should we add /list suffix here to maintain consistency across all list apis?

Reply:
by the rest api guidelines, the / endpoint should provide the list and then to access the individual resource it should be retrieved using /{customer_id}